### ISSUE TITLE
privacyIDEA easy setup and standalone

### ISSUE THREAD

**Issue by cornelinux** [privacyidea/privacyidea#1093]
# Objectives

1. We want to allow an easy setup of privacyIDEA, if privacyIDEA is not installed with a meta package on Ubuntu or RHEL, which sets up everything.
2. We want to allow to use privacyIDEA without the need of running a webserver. 

# Ideas

* Enhance the ``pi-manage`` script to allow creation of ``pi.cfg``.
* provide a script to verify OTP
* easy setup of pi.cfg without a webserver, but with sqlite.

**Comment by fredreichbier** [comment]
As a starting point, I reused some code from the standalone PR and threw together [1093/pi-bootstrap](https://github.com/privacyidea/privacyidea/tree/1093/pi-bootstrap), which adds two new commands to ``pi-manage``:
```shell
$ pi-manage bootstrap generate_config -b sqlite:////var/lib/privacyidea.db
# the database string is required. creates /etc/privacyidea/pi.cfg
$ cat /etc/privacyidea/pi.cfg
import logging
PI_ENCFILE = '/etc/privacyidea/encKey'
PI_AUDIT_KEY_PRIVATE = '/etc/privacyidea/private.pem'
PI_AUDIT_KEY_PUBLIC = '/etc/privacyidea/public.pem'
PI_LOGFILE = '/var/log/privacyidea.log'
PI_LOGLEVEL = logging.INFO
SQLALCHEMY_DATABASE_URI = u'sqlite:////var/lib/privacyidea.db'
SECRET_KEY = ' Q\xe6\x7f\x00u4\x80\xbd\xf1\xef\xc0h\n\r\x0f.8&\x87U\x12\x92J'
PI_PEPPER = '9qeudR3BbthANG8YvGGOx4OG'
$ ./pi-manage bootstrap setup
# runs create_enckey, create_audit_keys and createdb
```
... could be useful to set up privacyIDEA from pip, as you described. Hmm, but I'm currently not sure how we could nicely integrate the standalone/sqlite pi.cfg in there. What do you think?

**Comment by cornelinux** [comment]
Closed by #1090
